shortnames/shortnames.conf

```diff
@@ -1,13 +1,15 @@
 [shortnames]
-  "granite" = "huggingface://ibm-granite/granite-8b-code-instruct-GGUF/granite-8b-code-instruct.Q4_K_M.gguf"
+  "granite" = "granite3.1-dense"
```

Comment:
If we want to pull from ollama, this should probably be
ollama://granite3.1-dense
The rest below should be the same.

If user changes default transport these would be attempted to be pulled from OCI or HF.

Reply:
Done